### PR TITLE
hybridbar: init at 0.4.9

### PR DESCRIPTION
**Pull request by ixi2101**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -941,11 +941,7 @@
     githubId = 16022854;
     matrix = "@albertodvp:matrix.org";
     name = "Alberto Fanton";
-    keys = [
```
Comment (AndersonTorres) [inline maintainers/maintainer-list.nix:944]: Undo the formatting.

File: pkgs/by-name/hy/HybridBar/package.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  src = fetchFromGitHub {
+    url = "https://github.com/vars1ty/HybridBar/archive/refs/tags/0.4.9.tar.gz";
+    owner = "vars1ty";
+    repo = "HybridBar";
+    rev = "v0.4.9";
+    sha256 = "sha256-e9QVDDN8AtCZYuYqef1rzLJ0mklaKXzxgj+ZqGrSYEY=";
+  };
+  cargoHash = "sha256-50Go2AJQqgG2chc70KqKJexffD1rpObXr85ZPAIqc/U=";
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [
```
Comment (AndersonTorres) [inline pkgs/by-name/hy/HybridBar/package.nix:27]:
```suggestion
  
  cargoHash = "sha256-50Go2AJQqgG2chc70KqKJexffD1rpObXr85ZPAIqc/U=";
  
  nativeBuildInputs = [ pkg-config ];
  
  buildInputs = [
```

File: pkgs/by-name/hy/HybridBar/package.nix [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+rustPlatform.buildRustPackage rec {
+  pname = "HybridBar";
+  version = "0.4.9";
+
+  src = fetchFromGitHub {
+    url = "https://github.com/vars1ty/HybridBar/archive/refs/tags/0.4.9.tar.gz";
+    owner = "vars1ty";
+    repo = "HybridBar";
+    rev = "v0.4.9";
+    sha256 = "sha256-e9QVDDN8AtCZYuYqef1rzLJ0mklaKXzxgj+ZqGrSYEY=";
```
Comment (AndersonTorres) [inline pkgs/by-name/hy/HybridBar/package.nix:23]:
```suggestion
    hash = "sha256-e9QVDDN8AtCZYuYqef1rzLJ0mklaKXzxgj+ZqGrSYEY=";
```

File: pkgs/by-name/hy/HybridBar/package.nix [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+rustPlatform.buildRustPackage rec {
+  pname = "HybridBar";
+  version = "0.4.9";
+
+  src = fetchFromGitHub {
+    url = "https://github.com/vars1ty/HybridBar/archive/refs/tags/0.4.9.tar.gz";
+    owner = "vars1ty";
+    repo = "HybridBar";
+    rev = "v0.4.9";
```
Comment (AndersonTorres) [inline pkgs/by-name/hy/HybridBar/package.nix:22]: ??

```suggestion
    rev = "v${version}";
```

File: pkgs/by-name/hy/HybridBar/package.nix [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [
+    gtk-layer-shell
+    gtk3
+    glib
+    pango
+    cairo
+  ];
+
+  meta = with lib; {
```
Comment (AndersonTorres) [inline pkgs/by-name/hy/HybridBar/package.nix:35]:
```suggestion
  meta = {
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Swap the commits

**Review by AndersonTorres — APPROVED** [review]

**Comment by AndersonTorres** [comment]
Extra

rename the package directory to lowercase

File: pkgs/by-name/hy/HybridBar/package.nix [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  buildInputs = [
+    gtk-layer-shell
+    gtk3
+    glib
+    pango
+    cairo
+  ];
+
+  meta = {
+    description = "A status bar focused on wlroots Wayland compositors";
```
Comment (Aleksanaa) [inline pkgs/by-name/hy/HybridBar/package.nix:38]:
```suggestion
    description = "Status bar focused on wlroots Wayland compositors";
```

**Comment by Aleksanaa** [comment]
And reword commit message to lower case

File: pkgs/by-name/hy/HybridBar/package.nix (hunk 6 above)
Comment (ixi2101) [reply pkgs/by-name/hy/HybridBar/package.nix:38]: Description taken from the HybridBar repo itself. 
Not changing

Comment (AndersonTorres) [reply pkgs/by-name/hy/HybridBar/package.nix]: Nonetheless articles at the beginning should be removed.

**Review by AndersonTorres — APPROVED** [review]

**Review by Aleksanaa — APPROVED** [review]

File: pkgs/by-name/hy/hybridbar/package.nix [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "HybridBar";
+  version = "0.4.9";
+
+  src = fetchFromGitHub {
+    owner = "vars1ty";
+    repo = "HybridBar";
+    rev = "v${version}";
```
Comment (Aleksanaa) [inline pkgs/by-name/hy/hybridbar/package.nix:21]:
```suggestion
    rev = "refs/tags/${version}";
```

There's no `v` prefix at all.

**Pull request closed without merging**